tests/core/recipes/espresso_recipes/test_bands.py

```diff
@@ -14,9 +14,12 @@
 
 DATA_DIR = Path(__file__).parent / "data"
 
+DEFAULT_PARALLEL_INFO = {"binary": "mpirun", "-np": 2}
```

Comment:
Unfortunately, we don't want to require that the user has the `mpirun` binary on their machine. For instance, a clean install on my WSL machine doesn't have it.

I'd suggest doing something like:

```python
from shutil import which

if which("mpirun"):
    DEFAULT_PARALLEL_INFO = {...}
else:
    DEFAULT_PARALLEL_INFO = {}
```